Makefile

```diff
@@ -12,6 +12,16 @@ vanilla: ## Builds latest stable kernel, unpatched
 		--return \
 		$(OUT)
 
+.PHONY: tiny-6.6
+tiny-6.6: OUT:=$(SCRIPT_OUTPUT_PREFIX)-tiny-6.6.$(SCRIPT_OUTPUT_EXT)
```

Comment:
We should have CI run this instead of "vanilla"

Also consider having CI run `make tinyconfig` instead of needing to commit it.